packages/account/src/Types/common-prop.type.ts

```diff
@@ -55,8 +58,8 @@ export type TApiContext = {
     footer_ref: Element | DocumentFragment | undefined;
     overlay_ref:
         | ((...args: unknown[]) => unknown)
-        | import('prop-types').InferProps<{
-              current: import('prop-types').Requireable<unknown>;
+        | InferProps<{
+              current: Requireable<unknown>;
           }>;
     toggleOverlay: () => void;
 };
```

Comment:
why are we deleting below file

Reply:
@amina-deriv He just imported the files at the top the file, he didn't remove anything

Reply:
i mean this file packages/account/src/Types/common-prop.types.ts

Reply:
The content of that file already exists in another file. Hence removed it